src/main/java/mServer/crawler/sender/br/BrWebAccessHelper.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * BrWebAccessHelper.java
+ * 
+ * Projekt    : MServer
+ * erstellt am: 07.10.2017
+ * Autor      : Sascha
+ * 
+ * (c) 2017 by Sascha Wiegandt
+ */
+package mServer.crawler.sender.br;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)